connectors/src/connectors/zendesk/temporal/activities.ts

```diff
@@ -347,6 +347,7 @@ export async function syncZendeskArticleBatchActivity({
     brandId: category.brandId,
     connectorId,
   });
+  zendeskApiClient.config.subdomain = brandSubdomain;
```

Comment:
what does this do?

Reply:
it scopes the client to a brand, it is usually hidden in `changeZendeskClientSubdomain` but here it's super awkward because we need the subdomain alone as well
I'll change the function `changeZendeskClientSubdomain` to return the subdomain and hide this mysterious config change